calibre-web.json

```
@@ -0,0 +1,26 @@
+{
+    "name": "Calibre-Web",
+    "plugin_schema": "2",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/entuerto/iocage-plugin-calibre-web.git",
+    "pkgs": [
+        "python37",
+        "py37-pip",
+        "py37-setuptools",
+        "py37-sqlite3"
+    ],
+    "properties": {
+        "dhcp": 1
+    },
+    "packagesite": "http://pkg.FreeBSD.org/${ABI}/latest",
+    "fingerprints": {
+        "plugin-default": [
+            {
+                "function": "sha256",
+                "fingerprint": "226efd3a126fb86e71d60a37353d17f57af816d1c7ecad0623c21f0bf73eb0c7" // ???
```

```suggestion
                "fingerprint": "b0170035af3acc5f3f3ae1859dc717101b4e6c1d0a794ad554928ca0cbb2f438"
```

Should probably be the correct trusted fingerprint (you can check the other plugins have the same one for the latest packagesite)